custom_dc/index.md

```diff
@@ -95,3 +95,13 @@ The cost of running a site on Google Cloud Platform depends on the size of your
 1. Build a custom image, upload it to the Google Cloud Artifact Registry and create a Cloud Run service to run the site. See [Deploy services to Google Cloud](deploy_cloud.md)
 1. Launch and productionize your site for external traffic. See [Launch your Data Commons](/custom_dc/launch_cloud.html).
 1. For future updates and launches, continue to make UI and data changes locally, before deploying the changes to GCP.
+
+## Send feedback {#feedback}
+
+We use [https://issuetracker.google.com](https://issuetracker.google.com){: target="_blank"} to track bugs and feature requests. All tickets are is publicly viewable by all users and partners of Data Commons.
+
+- To make a feature request, please see if an [existing request]https://issuetracker.google.com/issues?q=componentid:1659535%2B%20type:feature_request){: target="_blank"} already exists. If it does, upvote and subscribe to it. If not, open a new feature request[here](https://issuetracker.google.com/issues/new?component=1659535){: target="_blank"}.
```

Comment:
Add `&template=2053233` to the link?

Reply:
I don't think it's needed -- I think it gets added automatically.

Reply:
Ah okay. Maybe slight preference for adding it anyway, just in case we want to change the default template in the future? But fine either way.

Reply:
That'd be my preference as well to give us the flexibility to change defaults.

Reply:
OK, I'll add it back.